### PR TITLE
Add missing "errors" sections for `PDO` and `PDOStatement` methods

### PR DESCRIPTION
**Pull request by phansys**

### CODE REVIEW

File: reference/pdo/pdo/exec.xml [hunk 1]
```diff
@@ -71,6 +71,19 @@ $db->exec() or die(print_r($db->errorInfo(), true)); // incorrect
...
+  &reftitle.errors;
+  <listitem>
+   <para>
+    Emits an error with level <constant>E_WARNING</constant> if the attribute <constant>PDO::ATTR_ERRMODE</constant> is set
+    to <constant>PDO::ERRMODE_WARNING</constant>.
+   </para>
+   <para>
+    Throws a <classname>PDOException</classname> if the attribute <constant>PDO::ATTR_ERRMODE</constant>
+    is set to <constant>PDO::ERRMODE_EXCEPTION</constant>.
+  </para>
```
Comment (Girgias) [inline reference/pdo/pdo/exec.xml:84]: Two things, make this an XML entity in language snippets to not need to copy paste this over and over.

Secondly:
```suggestion
   <para>
    Emits an error with level <constant>E_WARNING</constant> if the attribute <constant>PDO::ATTR_ERRMODE</constant> is set
    to <constant>PDO::ERRMODE_WARNING</constant>.
   </para>
   <para>
    Throws a <classname>PDOException</classname> if the attribute <constant>PDO::ATTR_ERRMODE</constant>
    is set to <constant>PDO::ERRMODE_EXCEPTION</constant>.
  </para>
```
this listitem tag makes no sense.

Comment (phansys) [reply reference/pdo/pdo/exec.xml]: Thanks for the hints.
I'm not familiar with the structure nor the entities, so please let me know if there is some additional change that must be made.

File: language-snippets.ent [hunk 2]
```diff
@@ -1814,6 +1814,17 @@ driver, if your code can run against multiple drivers.</simpara>'>
...
 
+<!-- PDO errors -->
+
+<!ENTITY pdo.errors '<para>
+Emits an error with level <constant>E_WARNING</constant> if the attribute <constant>PDO::ATTR_ERRMODE</constant> is set
+to <constant>PDO::ERRMODE_WARNING</constant>.
+</para>
+<para>
+Throws a <classname>PDOException</classname> if the attribute <constant>PDO::ATTR_ERRMODE</constant>
+is set to <constant>PDO::ERRMODE_EXCEPTION</constant>.
```
Comment (Girgias) [inline language-snippets.ent:1825]: Wording improvement:
```suggestion
 Emits an <constant>E_WARNING</constant> level error if the 
 <constant>PDO::ATTR_ERRMODE</constant> attribute is set to
 <constant>PDO::ERRMODE_WARNING</constant>.
</para>
<para>
 Throws a <classname>PDOException</classname> if the
 <constant>PDO::ATTR_ERRMODE</constant>  attribute is set to
 <constant>PDO::ERRMODE_EXCEPTION</constant>.
```

File: language-snippets.ent [hunk 3]
```diff
@@ -1814,6 +1814,19 @@ driver, if your code can run against multiple drivers.</simpara>'>
 
 <!ENTITY pdo.errors.exception-not-errmode '<note xmlns="http://docbook.org/ns/docbook"><simpara>An exception is raised even when the <constant>PDO::ATTR_ERRMODE</constant> attribute is not <constant>PDO::ERRMODE_EXCEPTION</constant>.</simpara></note>'>
 
+<!-- PDO errors -->
+
+<!ENTITY pdo.errors '<para>
```
Comment (Girgias) [inline language-snippets.ent:1819]:
```suggestion
<!ENTITY pdo.errors '<para xmlns="http://docbook.org/ns/docbook">
```

File: language-snippets.ent [hunk 4]
```diff
@@ -1821,6 +1821,17 @@ driver, if your code can run against multiple drivers.</simpara>'>
 
 <!ENTITY pdo.errors.exception-not-errmode '<note xmlns="http://docbook.org/ns/docbook"><simpara>An exception is raised even when the <constant>PDO::ATTR_ERRMODE</constant> attribute is not <constant>PDO::ERRMODE_EXCEPTION</constant>.</simpara></note>'>
 
+<!-- PDO errors -->
+
+<!ENTITY pdo.errors '<para xmlns="http://docbook.org/ns/docbook">
+Emits an error with level <constant>E_WARNING</constant> if the attribute <constant>PDO::ATTR_ERRMODE</constant> is set
+to <constant>PDO::ERRMODE_WARNING</constant>.
+</para>
+<para>
```
Comment (Girgias) [inline language-snippets.ent:1830]:
```suggestion
<para xmlns="http://docbook.org/ns/docbook">
```

**Comment by Girgias** [comment]
Thank you!

**Pull request merged**